html/includes/print-vm.inc.php

```diff
@@ -10,10 +10,10 @@
 
 echo '</td>';
 
-if ($vm['vmwVmState'] == 'powered off') {
-    echo '<td class="list"><span style="min-width:40px; display:inline-block;" class="label label-default">OFF</span></td>';
-} else {
+if ($vm['vmwVmState'] == 'running' || $vm['vmwVmState'] == 'powered on') {
```

Comment:
How do we know that this won't affect vmware based virtual machines?

I don't have any of either to test to be sure.

Reply:
As far as I can tell VMWare returns either 'powered on' or 'powered off'. I'll spin up esxi in a VM and do some testing once I've made some changes to this PR.